Casks/ableton-live-intro.rb

```diff
@@ -1,8 +1,11 @@
 cask "ableton-live-intro" do
   arch = "intel"
```

Comment:
This should be replaced with an `arch` stanza (this also applies to the other `ableton-live-*` casks):

```suggestion
  arch arm: "universal", intel: "intel"
```

Reply:
Wouldn't `universal` work for both?

Reply:
The `on_high_sierra` stanza has an arch of `intel` so I don't think so?